server/src/jest.config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "globals": {
+    "ts-jest": {
+      "tsconfig": "./src/tsconfig.app.json"
+    }
+  },
+  "moduleFileExtensions": ["js", "json", "ts"],
+  "rootDir": ".",
+  "testEnvironment": "node",
+  "testRegex": ".spec.ts$",
+  "transform": {
+    "^.+\\.(t|j)s$": "ts-jest"
+  }
+}
```

Comment:
basically, being more explicit about what should be included as unit tests and not include other tests